### PR TITLE
Oxidation anonymous cmds

### LINKED ISSUE #3871: `client_connection` provide a way to send anonymous

### LINKED ISSUE #3877: Translate anonymous cmds (the commands in `cmds.py`)

### PR DESCRIPTION
**Pull request by TimeEngineer**
# What has changed ?

- [ ] Does this PR affect the User ?

closes #3871
closes #3877

### CODE REVIEW

**Review by FirelightFlagboy — APPROVED** [review]

File: oxidation/libparsec/crates/types/src/addr.rs [hunk 1]
```diff
@@ -803,6 +803,10 @@ impl BackendInvitationAddr {
             root_verify_key,
         )
     }
+
+    pub fn to_invited_url(&self) -> Url {
```
Comment (mmmarcos) [inline oxidation/libparsec/crates/types/src/addr.rs:807]:
```suggestion
    /// Return an [Url] that points to the server endpoint for invited commands.
    pub fn to_invited_url(&self) -> Url {
```

**Review by mmmarcos — CHANGES_REQUESTED** [review]

File: oxidation/libparsec/crates/types/src/addr.rs (hunk 1 above)
Comment (TimeEngineer) [reply oxidation/libparsec/crates/types/src/addr.rs:808]: Done

**Review by mmmarcos — APPROVED** [review]
LGTM

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: HISTORY.rst [hunk 2]
```diff
@@ -119,7 +119,7 @@ Features
   <https://github.com/Scille/parsec-cloud/issues/3056>`__)
 * Updated the file link not found error message with the name of the file
   (`#3122 <https://github.com/Scille/parsec-cloud/issues/3122>`__)
-* Added new context menu option to generate file links that point to a specific
+* Added new context menu option to generate file links that points to a specific
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply HISTORY.rst:122]: If you want to include this change in this PR, extract that change to a specific commit that only modify this file

File: parsec/core/backend_connection/anonymous.py [hunk 3]
```diff
@@ -90,21 +95,31 @@ async def pki_enrollment_submit(
...
 ) -> PkiEnrollmentSubmitRep:
-    rep = cast(
-        PkiEnrollmentSubmitRep,
-        await _anonymous_cmd(
-            serializer=pki_enrollment_submit_serializer,
-            cmd="pki_enrollment_submit",
-            addr=addr,
-            organization_id=addr.organization_id,
+    if FEATURE_FLAGS["UNSTABLE_OXIDIZED_CLIENT_CONNECTION"]:
+        rep = await AnonymousCmds(addr).pki_enrollment_submit(
```
Comment (FirelightFlagboy) [inline parsec/core/backend_connection/anonymous.py:99]: Should we re-build every time the HTTP client ?

File: parsec/core/backend_connection/anonymous.py [hunk 4]
```diff
@@ -5,6 +5,7 @@
 
 from structlog import get_logger
 
+from parsec import FEATURE_FLAGS
```
[the remark this answers is not in the document]
Comment (TimeEngineer) [reply parsec/core/backend_connection/anonymous.py:8]: Done

File: parsec/core/backend_connection/anonymous.py [hunk 5]
```diff
@@ -90,21 +91,34 @@ async def pki_enrollment_submit(
...
-        await _anonymous_cmd(
-            serializer=pki_enrollment_submit_serializer,
-            cmd="pki_enrollment_submit",
-            addr=addr,
-            organization_id=addr.organization_id,
+    if FEATURE_FLAGS["UNSTABLE_OXIDIZED_CLIENT_CONNECTION"]:
+        from parsec._parsec import AnonymousCmds
+
+        client = AnonymousCmds(addr)
+        rep = await client.pki_enrollment_submit(
```
[the remark this answers is not in the document]
Comment (TimeEngineer) [reply parsec/core/backend_connection/anonymous.py]: Done

File: parsec/core/backend_connection/anonymous.py [hunk 6]
```diff
@@ -120,16 +134,22 @@ async def pki_enrollment_submit(
...
-            addr=addr,
-            organization_id=addr.organization_id,
-            enrollment_id=enrollment_id,
-        ),
-    )
+    if FEATURE_FLAGS["UNSTABLE_OXIDIZED_CLIENT_CONNECTION"]:
+        from parsec._parsec import AnonymousCmds
+
+        client = AnonymousCmds(addr)
+        rep = await client.pki_enrollment_info(enrollment_id=enrollment_id)
```
[the remark this answers is not in the document]
Comment (TimeEngineer) [reply parsec/core/backend_connection/anonymous.py]: Done

File: parsec/core/backend_connection/anonymous.py [hunk 7]
```diff
@@ -151,22 +171,36 @@ async def organization_bootstrap(
...
-        await _anonymous_cmd(
-            organization_bootstrap_serializer,
-            cmd="organization_bootstrap",
-            addr=addr,
-            organization_id=addr.organization_id,
+    if FEATURE_FLAGS["UNSTABLE_OXIDIZED_CLIENT_CONNECTION"]:
+        from parsec._parsec import AnonymousCmds
+
+        client = AnonymousCmds(addr)
+        rep = await client.organization_bootstrap(
```
[the remark this answers is not in the document]
Comment (TimeEngineer) [reply parsec/core/backend_connection/anonymous.py]: Done

File: parsec/core/backend_connection/anonymous.py (hunk 3 above)
Comment (TimeEngineer) [reply parsec/core/backend_connection/anonymous.py:99]: It's the same in python, I understand that is called very rarely so we don't really care, it's like reqwest::get, where we don't keep the client

File: HISTORY.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (TimeEngineer) [reply HISTORY.rst:122]: I will rollback this change

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**